### PR TITLE
New new search

### PR DESCRIPTION
**Pull request by Dantemss**
Compatibility with OSU 4.2.0, plus some better styling for the dev login.

### CODE REVIEW

**Comment by Dantemss** [comment]
Needs an update for Rails 4.2 and Roar 1.0

**Comment by Dantemss** [comment]
Still uses Roar < 1.0 but should be good to merge. I want to write some tests for the representers before we update Roar.

File: app/routines/openstax/accounts/search_local_accounts.rb [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+      }
+
+      lev_routine
+
+      uses_routine OSU::SearchAndOrganizeRelation, as: :search,
+                   translations: { outputs: { type: :verbatim } }
+
+      def exec(*args)
+        params = args.last.is_a?(Hash) ? args.pop : {}
+        params[:q] ||= args[0]
```
Comment (jpslav) [inline app/routines/openstax/accounts/search_local_accounts.rb]: These are cryptic variables.  This way because of code in OSU?

Comment (Dantemss) [inline app/routines/openstax/accounts/search_local_accounts.rb]: query, order_by(s?), per_page and page would also work, OSU accepts both

**Pull request merged**